tools/src/commands/GenerateBareApp.ts

```diff
@@ -0,0 +1,133 @@
+import path from 'path';
+import fs from 'fs-extra';
+import { Command } from '@expo/commander';
+import spawnAsync from '@expo/spawn-async';
+import { runExpoCliAsync } from '../ExpoCLI';
+
+type GenerateBareAppOptions = {
+  template: string;
+  clean?: boolean;
+  outDir?: string;
+};
+
+async function action(appName: string, packageNames: string[], options: GenerateBareAppOptions) {
+  function symlinkPackageManually(packageName: string) {
+    const packagePath = path.resolve(projectDir, 'node_modules', packageName);
+    if (fs.existsSync(packagePath)) {
+      fs.rmdirSync(path.resolve(projectDir, 'node_modules', packageName), { recursive: true });
+    }
+
+    fs.symlinkSync(path.resolve(repoRoot, 'packages', packageName), packagePath);
+  }
+
+  // TODO:
+  // if appName === ''
+  // if packageNames.length === 0
+  // configure autolinking to exclude dev-client / expo update packages
+
+  const { clean, outDir = 'bare-apps', template = 'expo-template-bare-minimum' } = options;
+
+  const repoRoot = path.resolve(__dirname, '../../../');
+  const projectsDir = path.resolve(process.cwd(), outDir);
+  const projectDir = path.resolve(process.cwd(), projectsDir, appName);
+
+  if (!fs.existsSync(projectsDir)) {
+    fs.mkdirSync(projectsDir);
+  }
+
+  if (clean) {
+    if (fs.existsSync(projectDir)) {
+      fs.rmdirSync(projectDir, { recursive: true });
+    }
+  }
```

Comment:
With `fs.remove` from `fs-extra` you don't have to check if the file exists. See https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/remove.md
```suggestion
  if (clean) {
    await fs.remove(projectDir);
  }
```

Reply:
🎉 TIL! thank you